assessment.sql

```diff
@@ -4,25 +4,49 @@ CREATE DATABASE "sql-crud-assessment";
 \c "sql-crud-assessment"
 
 -- Question 1. Create a table called `licensees` to hold data in columns surname,given_name,gender,height,weight,dob,eye_color,hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
+CREATE TABLE licensees (
+  id SERIAL PRIMARY KEY,
+  surname CHARACTER VARYING,
+  given_name CHARACTER VARYING,
+  gender CHARACTER VARYING(1),
+  height INTEGER,
+  weight INTEGER,
+  dob DATE,
+  eye_color CHARACTER VARYING,
+  hair_color CHARACTER VARYING
+);
+
 
 -- your code here
 
 -- Question 2. Add `Bishop, Caren, f, 63, 132, 1943-09-26, Brown, Black` into the table of licensees.
-
+INSERT INTO licensees(surname,given_name,gender,height,weight,dob,eye_color,hair_color)
+VALUES('Bishop', 'Caren', 'f', 63, 132, '1943-09-26', 'Brown', 'Black')
+;
 -- your code here
 
 -- Question 3. Load the data from `data/licensees.csv` into the `licensees` table.
 
+\copy licensees(surname,given_name,gender,height,weight,dob,eye_color,hair_color) FROM 'data/licensees.csv' WITH (HEADER true, FORMAT csv);
 -- your code here
 
 -- Question 4. Write a query to get all attributes of licensees with Hazel eye color and Brown or Black hair color.
+SELECT * from licensees WHERE eye_color = 'Hazel', hair_color = 'Brown' OR hair_color = 'Black'
+;
 
 -- your code here
 
 -- Question 5. Make each of Marilynn Escobar and Chris Whaley one inch shorter. Look them up by their names.
 
+UPDATE licensees
+SET height = height - 1
+WHERE (given_name = 'Marilynn' AND surname = 'Escobar') AND (given_name = 'Chris' AND surname = 'Whaley')
+;
```

Comment:
Very close. That middle `AND` causes this to update no rows because the two subexpressions contradict each other. Never will you find a licensee in the table whose name is simultaneously "Marilynn Escobar" and "Chris Whaley". 3/4